src/workflows/height-depthmapmultiartifactlatefusion-workflow.json

```diff
@@ -1,10 +1,10 @@
 {
-    "name": "q3-depthmapmultiartifactlatefusion-plaincnn-height-95k",
+    "name": "q3-depthmapmultiartifactlatefusion-height",
     "version": "run-06_01",
     "result_format": "dict",
     "result_binding": "scan",
     "data":{
         "input_format":"application/zip",
-        "model":"q3-depthmapmultiartifactlatefusion-plaincnn-height-95k"
+        "model":"q3-depthmapmultiartifactlatefusion-height"
```

Comment:
model name should be "q3-depthmapmultiartifactlatefusion-plaincnn-height-95k"

Reply:
The [experiment](https://ml.azure.com/experiments/id/f42711e3-7642-426d-ba53-a616a07e076b?wsid=/subscriptions/9b5bbfae-d5d1-4aae-a2ca-75159c0c887d/resourceGroups/cgm-ml-prod-we-rg/providers/Microsoft.MachineLearningServices/workspaces/cgm-ml-prod-we-azml&tid=3a27c573-ec1a-4734-9cd3-3208af51794b) name should probably be used (because we are referencing the run).

![image](https://user-images.githubusercontent.com/1809832/112603229-fffa0e80-8e14-11eb-91da-390ae5252bda.png)

I also suggests the name `q3-depthmapmultiartifactlatefusion-plaincnn-height-95k`. Therefore I agree with @gangooteli . For other workflows, we can also take the AzureML experiment name

Reply:
Thank You.
@gangooteli  @markus-hinsche